### DISCUSSION TITLE
Windows live AI proctoring with non-LMS solutions

### DISCUSSION THREAD

**Discussion by bryndin32** [SafeExamBrowser/seb-server#47] [Q&A]
Hi! First of all thank you for the great product you make!

Our team is interested in integrating SEB and its AI live proctoring features with non-LMS E-assessment solution based on low-code BPMS. We've succeded in running an exam session with live proctoring using iOS application. 
However, unlike the iOS app, the SEB server (and the Windows app connected to it) allow to use AI live proctoring only with exams imported from LMS (Moodle, etc.).  Is there any way to use SEB server with Windows application for live proctoring of non-LMS exams?

Thanks!

**Comment by dbuechel** [comment]
Currently not, as far as I am informed. But @anhefti may have further information or a roadmap regarding integration of non-LMS exams.

**Reply by bryndin32** [reply]
@dbuechel Thank you very much for the reply!

**Comment by anhefti** [comment]
Since SEB Server and its functionality is closely bound to LMS courses or exams (time-frame, course/exam content, user-login) there is yet no possibility and also no plan to use SEB Server with non-LMS exams

**Reply by bryndin32** [reply]
@anhefti Thanks for the information!
